tests/test_settings.py

```diff
@@ -117,3 +117,24 @@ def test_get_callback_function_with_non_callable_string(self, import_string_mock
     def test_get_callback_function_(self):
         with self.assertRaises(ImportError):
             get_callback_function("DJSTRIPE_TEST_CALLBACK")
+
+
+class TestGetApiVersion(TestCase):
+
+    @override_settings(DJSTRIPE_API_VERSION=None)
+    def test_with_default(self):
+        self.assertEquals('2017-02-14', get_api_version())
```

Comment:
All this does is require updating the default in two places instead of one. Can you strike it?